integration_tests/tests/apply/withdrawApplication.cy.ts

```diff
@@ -43,7 +44,7 @@ context('Withdraw Application', () => {
     const withdrawConfirmationPage = Page.verifyOnPage(WithdrawApplicationPage)
 
     // When I choose a reason and click submit
-    withdrawConfirmationPage.completeForm(withdrawalReason)
+    withdrawConfirmationPage.completeForm(withdrawalReason, reasonDescription)
```

Comment:
Maybe not worth having two integration tests for one conditional input, but is it worth noting somewhere that this hardcodes in an assumption that the user is selecting `'other'`, and that the reason shouldn't be included if not?

Reply:
Sounds like a job for type inference: 
`completeForm<T extends WithdrawalReason>(withdrawalReason: T, otherReason: T extends 'other' ? string : never) `